packages/commerce-sdk-react/src/hooks/ShopperProducts/query.ts

```diff
@@ -33,6 +34,9 @@ function useProducts(
     options?: UseQueryOptions<DataType<Client['getProducts']> | Response, Error>
 ): UseQueryResult<DataType<Client['getProducts']> | Response, Error> {
     const {headers, rawResponse, ...parameters} = arg
+    const {locale, currency} = useConfig()
+    parameters.locale = parameters.locale || locale
+    parameters.currency = parameters.currency || currency
```

Comment:
I see.. we haven't been using and passing in the `locale` until now 👍 

This change is worthy to share with everyone in the team. For every hooks we're working on, we'll need to make sure to pass in the locale, if it's supported for that particular API endpoint.